runtime/src/main/java/com/datastax/oss/quarkus/runtime/internal/context/QuarkusDriverContext.java

```diff
@@ -55,4 +62,10 @@ protected NettyOptions buildNettyOptions() {
       return super.buildNettyOptions();
     }
   }
+
+  @NonNull
+  @Override
+  public String getSessionName() {
+    return CASSANDRA_METRICS_PREFIX + "." + super.getSessionName();
+  }
```

Comment:
Is the goal of this change to distinguish java-driver sessions from Quarkus sessions?  Or are you looking to accomplish something else here?

Reply:
yes, we want all driver metrics to be under the specific prefix: `cassandra` in this case